src/view/constants.ts

```diff
@@ -42,7 +42,7 @@ export const CONSTANTS = {
         NUMERIC_SIX: "Digit6",
         NUMERIC_SEVEN: "Digit7",
     },
-    NO_FILES_AVAILABLE: "Choose a .py file to run on the Simulator",
+    NO_FILES_AVAILABLE: "Save a .py file to run on the Simulator",
```

Comment:
Did you test the behavior when you press play and there are no python files open?
For me it crashes and gives out an error and then open a random file?
I think it makes more sense if we just pop up a warning or something similar.

Reply:
When I do it, it doesn't crash, but it does give the error and open a random file. This seems to be on dev as well and will be addressed as a separate issue